spec/load_fixtures_helper.rb

```diff
@@ -17,7 +17,7 @@
   end
   after do
     # TODO: danger - if there are objects in the test db we want to keep
-    HostSettings.storage_roots.each_value do |storage_root|
+    HostSettings.storage_roots.to_h.each_value do |storage_root|
```

Comment:
similarly, while in here, change to storage_root_location??

Reply:
Seems unrelated to my purpose.

Reply:
@ndushay You realize this line is completely broken as written, right?  Literally the block never executes now.  Example:
```ruby
> HostSettings.storage_roots.each_value { puts "XXX" }
=> nil
> HostSettings.storage_roots.to_h.each_value { puts "XXX" }
XXX
XXX
XXX
...
```